app/views/param/edit.scala.html

```diff
@@ -39,7 +39,7 @@ <h3 class="col-sm-5 control-label">@title</h3>
            <label for="outputName" class="col-sm-2 control-label">@Messages("parts.parameter.outputName")</label>
             <div class="row">
                 <div class="col-sm-5">
-                    <input name="outputName" type="text" value="@maybeParam.map(_.name).getOrElse("")" class="form-control validate[required]" />
+                    <input name="outputName" type="text" value="@maybeParam.fold("")(_.outputName)" class="form-control validate[required]" />
```

Comment:
I believe the `fold` and the previous `getOrElse` are both extraneous; Twirl automagically resolves optional types to a blank string. 

`@maybeParam.map(_.outputName)` should be enough :)